akka-http2-support/src/main/scala/akka/http/impl/engine/http2/framing/Http2FrameParsing.scala

```diff
@@ -162,5 +182,28 @@ private[http2] class Http2FrameParsing(shouldReadPreface: Boolean) extends ByteS
             UnknownFrameEvent(tpe, flags, streamId, payload.remainingData)
         }
       }
+
+      // FIXME this should be smarter, we need to somehow manage when we send back the ACK basically...
+      private def applySetting(s: SettingsFrame): SettingsFrame = {
+        s.settings.foreach {
+          case Setting(SettingIdentifier.SETTINGS_MAX_FRAME_SIZE, value) ⇒
+            settings.updateMaxOutFrameSize(value)
```

Comment:
I realize that this might be an artifact of the previous solution which would have communicated the peer intention with the renderer.